OpenIPSL/Electrical/Controls/PSAT/AVR/AVRTypeI.mo

```diff
@@ -1,34 +1,34 @@
 within OpenIPSL.Electrical.Controls.PSAT.AVR;
 model AVRTypeI "PSAT AVR Type 1"
 
-  Modelica.Blocks.Interfaces.RealInput v "Generator termminal voltage (pu)"
+  Modelica.Blocks.Interfaces.RealInput v "Generator termminal voltage [pu]"
     annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
         iconTransformation(extent={{-140,-80},{-100,-40}})));
-  Modelica.Blocks.Interfaces.RealOutput vf "Filed voltage (pu)" annotation (
+  Modelica.Blocks.Interfaces.RealOutput vf "Filed voltage [pu]" annotation (
       Placement(transformation(
         extent={{-10,-10},{10,10}},
         origin={110,0}), iconTransformation(extent={{100,-20},{140,20}})));
   Modelica.Blocks.Interfaces.RealInput vref
-    "Reference generator terminal voltage (pu)" annotation (Placement(
+    "Reference generator terminal voltage [pu]" annotation (Placement(
         transformation(extent={{-140,40},{-100,80}}), iconTransformation(extent
           ={{-140,40},{-100,80}})));
-  parameter Real vrmax=7.57 "Maximum regulator voltage (pu)";
-  parameter Real vrmin=0 "Minimum regulator voltage (pu)";
-  parameter Real K0=7.04 "Regulator gain, (pu/pu)";
-  parameter Real T1=6.67 "First pole (s)";
-  parameter Real T2=1 "First zero (s)";
-  parameter Real T3=1 "Second pole (s";
-  parameter Real T4=1 "Second pole (s)";
-  parameter Real Te=0.4 "Field circuit time constant (s)";
-  parameter Real Tr=0.05 "Measurement time constant (s)";
+  parameter SI.PerUnit vrmax=7.57 "Maximum regulator voltage";
+  parameter SI.PerUnit vrmin=0 "Minimum regulator voltage";
+  parameter Real K0=7.04 "Regulator gain, [pu/pu]";
```

Comment:
In the previous section the gain is defined as "parameter SI.PerUnit". Here it is "parameter Real". Does the difference depend on the model implementation?

Reply:
Since it is PSAT, I didn't want to look too much into it. And it was already defined as [pu/pu]. Should I change it to pu? What do you think? For me I will live it as it is.

Reply:
Ok leave it for now and let's review it later on